sdk/src/androidTest/java/com/mapbox/maps/ViewAnnotationTest.kt

```diff
@@ -317,8 +320,6 @@ class ViewAnnotationTest(
         assertEquals(mapboxMap.pixelForCoordinate(CAMERA_CENTER).y, secondView.translationY.toDouble(), ADMISSIBLE_ERROR_PX)
         assertArrayEquals(
           arrayOf(
-            Pair(firstView, true),
```

Comment:
Why does the actualVisibilityUpdateList changes here?

Reply:
with previous a bit chaotic algo with events coming from different observers we actually did add the view and immediately remove it actually resulting the listeners being hit and make view annotation show up - although I guess end user did not see that as view annotation was added for a moment behind the map (z-index).

now due to more systemic  approach we simply add one view annotation (second one as it makes the first one disappear).